app/src/main/res/layout/fragment_haushalt_bearbeiten_loeschen.xml

```diff
@@ -149,16 +149,15 @@
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintTop_toBottomOf="@+id/haushalteditTextdatum" />
```

Comment:
It's best practise to use snake_case within XML but it's not a must.

https://medium.com/mindorks/android-resource-naming-convention-42e4e8026614